frontend/src/tests/test_data.js

```diff
@@ -0,0 +1,258 @@
+import instructorsJSON from "./samples/instructors.json";
+import applicantsJSON from "./samples/applicants.json";
+import positionJSON from "./samples/positions.json";
+import wrong1positionsJSON from "./samples/wrong1positions.json";
```

Comment:
Be more descriptive with the file names. It's okay if they're really long as long as it's clear what the data is.